frontend/src/components/opportunity/OpportunityDocuments.tsx

```diff
@@ -0,0 +1,76 @@
+import { useTranslations } from "next-intl";
+import { Link, Table } from "@trussworks/react-uswds";
+
+interface OpportunityDocument {
+  opportunity_attachment_type: string;
+  file_name: string;
+  download_path: string;
+  updated_at: string;
+}
+
+interface OpportunityDocumentsProps {
+  documents: OpportunityDocument[];
+}
+
+const OpportunityDocuments = ({ documents }: OpportunityDocumentsProps) => {
+  const t = useTranslations("OpportunityListing.documents");
+
+  const documentTable = (documents_filtered: OpportunityDocument[]) => {
```

Comment:
seems to me like it'd be beneficial to split this into its own component. Potentially something that could be abstracted and reused somewhere else later, and making it its own thing not reliant on the wrapping component would be a good first step.

Reply:
You mean splitting up the table into it's own component? I did consider that too but I felt like it's be unnecessary wrapper around the existing trussworks table component since the code is like 99% the same its just the data that changes I figured just a function would serve that purpose for now, but open to a discussion about it. I guess I had the mentality of duplicate until it hurts, then abstract and taking this approach def didn't 'hurt'.

Reply:
Yeah, the future reusability may never really surface, but even without that, the only reason I can see to leave it nested inside the larger component like this is to close over the `t` function defined in the outer scope, which you can get around by calling the translation hook again in the new component. Other than that it is basically being used as its own component already so it seems clearer to me to define it that way to begin with. Happy to get on a call to discuss further